keyboards/dm9records/plaid/keymaps/gipsy-king/config.h

```diff
@@ -0,0 +1,18 @@
+/* Copyright 2019 Takuya Urakawa (dm9records.com)
```

Comment:
This file can be deleted.

Reply:
fixed, sorry for the inconvenience.
also spotted some missing keys (brackets) and fixed some alignment.